tests/test_reward_trainer.py

```diff
@@ -240,3 +240,146 @@ def test_tags(self):
                 model=self.model, args=training_args, processing_class=self.tokenizer, train_dataset=dummy_dataset
             )
             self.assertEqual(trainer.model.model_tags, trainer._tag_names)
+
+    def test_train_with_feedback(self):
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            # Create a dummy dataset with feedback
+            dummy_dataset_dict = {
```

Comment:
Ok now I get why you want a tiny version: the zen dataset misses two columns.
I'll think about the best way to do it